src/Entities/StrawberryJam/DashZipMover.cs

```diff
@@ -0,0 +1,411 @@
+﻿using Celeste.Mod.CommunalHelper;
+using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Monocle;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Celeste.Mod.CommunalHelper.Entities.StrawberryJam
+{
+    [CustomEntity("CommunalHelper/DashZipMover")]
+    public class DashZipMover : Solid
+    {
+        public const string game_dash_zip_mover_zip_mover = "event:/communalhelper/sj/game/dash_zip_mover/zip_mover";
```

Comment:
```suggestion
        public const string game_dash_zip_mover_zip_mover = "event:/CommunalHelperEvents/game/strawberryJam/game/dash_zip_mover/zip_mover";
```

Reply:
this constant could be moved to `CustomSFX`. also i have no idea if those exist in the FMOD bank